.travis.yml

```diff
@@ -1,14 +1,10 @@
 language: node_js
 node_js:
-  - '6'
+  - '12'
```

Comment:
Maybe add `- 14` too? It's the current LTS.

Reply:
It looks like we should move CI to Github Actions as with the other H3 repos.